include/fastboot.jinja2

```diff
@@ -3,7 +3,6 @@
 {% set use_context = true %}
 
 {% set DOCKER_IMAGE = DOCKER_IMAGE|default("debian:buster") %}
-{% set install_fastboot = install_fastboot|default(true) %}
```

Comment:
This belong to its own commit with the other 'install_fastboot' that gets removed.

Reply:
Ack will squash.